bin/setup_flat

```diff
@@ -72,7 +86,11 @@ container=forch-faucet-$inst
 container_inst=inst/$container/
 mkdir -p $container_inst/
 cp -r topo/$topo/faucet $container_inst/
+setup_flat_faucet_config $switches
+cp topo/common/faucet/ryu.conf $container_inst/faucet/
+cp topo/common/faucet/uniform.yaml $container_inst/faucet/
 cp -r topo/$topo/forch $container_inst/
+cp topo/common/forch/forch.yaml $container_inst/forch/
```

Comment:
common copy should go before $topo version

Reply:
Done